.github/workflows/scripts/scanner-offline-bundle-version-matrix.sh

```diff
@@ -0,0 +1,90 @@
+#!/usr/bin/env bash
+#
+# Print the mapping of StackRox releases to scanner vulnerability bundle schema
+# versions. One entry per line, separated by whitespace.
+
+set -euo pipefail
+
+DIR="$(cd "$(dirname "${BASH_SOURCE[0]}")" && pwd)"
+
+# GITHUB_URL_PATTERN points to vulnerability version file for a given branch/tag.
+GITHUB_URL_PATTERN="https://raw.githubusercontent.com/stackrox/stackrox/%s/scanner/VULNERABILITY_VERSION"
+
+RELEASE_VERSION_FILE="scanner/updater/version/RELEASE_VERSION"
+
+[ -f $RELEASE_VERSION_FILE ] || {
+    echo "error: release version file does not exist: $RELEASE_VERSION_FILE"
+    exit 1
+}
+
+# version_object() prints a version object in the matrix.
+version_object() {
+    local vuln_version=${1?:"missing positional argument 'vuln_version'"}
+    local releases=${2?:"missing positional argument 'vuln_version'"}
+    cat <<EOF
+  {
+    "vulnerability_version": "$vuln_version",
+    "supported_releases": "$releases"
+  }
+EOF
+
+}
+
+main() {
+    # Iterate over all tags that look like StackRox release tags, and output a JSON
+    # array.
+    local vuln_version
+    local vuln_version_url
+
+    declare -A releases
```

Comment:
fun fact: `-A` does not exist on darwin, so I can't run this script to test it. Though I can just read the CI logs, so I'll do that

Reply:
The `declare -A` is a shell's syntax, not the OS. It was introduced in bash around 2010, so maybe you have a very old bash installation? Or you're trying to run this with a different shell, but this script is specific for bash and should not be interpreted by other shell (unless that shell is guaranteed to be bash compatible).

Reply:
```
$ bash --version
GNU bash, version 3.2.57(1)-release (x86_64-apple-darwin22)
Copyright (C) 2007 Free Software Foundation, Inc.
```

seems like the default is OLD